lib/src/components/NxTabPanel/NxTabPanel.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import classnames from 'classnames';
+
+import { ActiveTabContext } from '../NxTabs/NxTabs';
+
+import { Props, propTypes } from './types';
+export { Props } from './types';
+
+const NxTabPanel = function NxTabPanelElement(props: Props) {
+  // Use React.useContext instead of importing useContext for jest to mock the value in the test
+  const activeTab = React.useContext(ActiveTabContext);
+  const {labelledBy, className, ...attrs} = props;
+
+  if (activeTab !== labelledBy) {
+    return null;
+  }
+
+  return (
+    <div role="tabpanel"
+         aria-expanded="true"
```

Comment:
I'm not sure this is right/ideal.  With our setup any tab panel that is present at all is visible, there isn't any collapse/expand concept that gives further distinction beyond the tab panel's presense.  ChromeVox for instance adds the word "expanded" to the verbal description of all tab panels due to this attribute, which isn't helpful.

Reply:
I'll remove this as it's unnecessary.

Reply:
> Looks like it's intended use is for a tab panel that allows multiple tabs to be open.

I think it's for things like accordions.